scripts/check_ecr_findings.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+# Set the repository name and image tag
+repository_name="$1"
+image_tag="$2"
+
+# Start the image scan
+aws ecr start-image-scan --repository-name "$repository_name" --image-id imageTag="$image_tag"
+
+# Wait for the scan to complete 
+echo "Waiting for scan to complete..."
+sleep 60 # To discuss sleep time with Clay 
```

Comment:
This fixed 60s sleep could be improved, as it has some shortcomings:
- If the scan takes _less_ than 60s, we wait the full 60s anyways
- If the scan takes _longer_ than 60s, it appears to exit with a failure

Rather than just a single 60s sleep, we should probably take an approach that loops a certain number of times, sleeping between each loop (with a max number of loops so that it doesn't run forever).

Reply:
Thanks for your feedback Clay. I will make the changes

Reply:
@ClayBenson94 I have made the changes and this PR is ready for approval.